docs/source/data-types/timeuuid.md

```diff
@@ -19,6 +19,41 @@ session
     .query("INSERT INTO keyspace.table (a) VALUES(?)", (to_insert,))
     .await?;
 
+// Read timeuuid from the table
+if let Some(rows) = session.query("SELECT a FROM keyspace.table", &[]).await?.rows {
+    for row in rows.into_typed::<(CqlTimeuuid,)>() {
+        let (timeuuid_value,): (CqlTimeuuid,) = row?;
+    }
+}
+# Ok(())
+# }
+```
+
+To use the Timeuuid on `uuid` crate, enable the feature `v1` in your crate using:
+
+```shell
+cargo add uuid -F v1
+```
+
+and now you're gonna be able to use the `uuid::v1` features: 
+
+```rust
+# extern crate uuid;
```

Comment:
I just pushed the example without the ignore. The problem is that Timeuuid uses the `v1` feature inside `uuid` package. But to create a example using that is not possible since the "extern crate" flag doesn't allow to add a feature, making the example fails. 

My first try was using the `ignore` flag to not pass in this validation, but didn't worked. 

The goal is to have something like this:
```suggestion
# extern crate uuid::v1; // Just a example
```

Reply:
> examples/Cargo.toml is the one used to configure deps for the docs. You can call cargo add uuid --dev -F v1 and the feature will be accessible from the docs.

> Ah, right. Then adding v1 to uuid's feature list in examples/Cargo.toml and removing this ignore should fix the problem.